### ISSUE TITLE
invalid-slot-type (org-jira-sdk-issue report string nil)

### ISSUE THREAD

**Issue by lvsv** [ahungry/org-jira#147]
I caught an error after today's merge when i had called `org-jira-get-issues`

```
signal (invalid-slot-type (org-jira-sdk-issue report string nil))
```
I saved `data` structure into variable and found empty  `(reporter)` list
```
  (customfield_16022)
  (customfield_18328)
  (customfield_18329)
  (reporter)
  (customfield_18324)
  (customfield_18325)
  (customfield_18326)
  (customfield_18327)
  (customfield_16935)
```

I curled this jira-issue, got the next one
```
   "reporter": {
      "self": "https://server.jira.local/rest/api/2/user?username=user1",
      "name": "user1",
      "key": "user1",
      "emailAddress": "?",
      "avatarUrls": {
        "48x48": "https://server.jira.local/secure/useravatar?avatarId=10143",
        "24x24": "https://server.jira.local/secure/useravatar?size=small&avatarId=10143",
        "16x16": "https://server.jira.local/secure/useravatar?size=xsmall&avatarId=10143",
        "32x32": "https://server.jira.local/secure/useravatar?size=medium&avatarId=10143"
      },
      "displayName": "user1",
      "active": false,
      "timeZone": "Europe/Moscow"
    },
```
`user1` is an unactive user with email address `?`

I think there are something wrong with parsing the `data` structure.

**Comment by lvsv** [comment]
Guys, I found the reason.

When we do `getIssuesFromJqlSearch` and jira-issue has inactive reporter then reporter will be null.
But if you do request to that jira-issue you will get correct reporter json.

So, sometimes reporter can be null.

**Comment by ahungry** [comment]
Thanks, I will push a commit that makes reporter optional.

**Comment by ahungry** [comment]
Let me know if this is still an issue by reopening if it isn't fixed in 15f1592
